src/Workspaces/VisualBasic/Portable/Extensions/ParenthesizedExpressionSyntaxExtensions.vb

```diff
@@ -68,6 +68,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions
                 Return True
             End If
 
+            ' Case:
+            '   ($"")
+            If expression.IsKind(SyntaxKind.InterpolatedStringExpression) Then
```

Comment:
Once we unified the syntax models, .Kind is just int-returning. This has the typecast in there since it's an extension method.